app/_layout.tsx

```diff
@@ -0,0 +1,15 @@
+import { Stack } from "expo-router";
+
+export default function Layout() {
+  return (
+    <Stack
+      screenOptions={{
+        headerStyle: {
+          backgroundColor: "#ED9E26",
+        },
+        headerTintColor: "#fff",
+        title: "Poezja",
```

Comment:
I wonder if we want to add this `useTheme` dummy hook right now or later, what do you think?